src/battle/overlay_12_0224E4FC.c

```diff
@@ -6478,3 +6488,435 @@ BOOL CheckStatusEffectsSubstitute(BATTLECONTEXT *ctx, int battlerId, int status)
+        (ctx->selfTurnData[ctx->battlerIdTarget].physicalDamage || ctx->selfTurnData[ctx->battlerIdTarget].specialDamage) &&
+        (ctx->unk_334.moveData[ctx->moveNoCur].unkB & 1)) {
+        *script = 216;
+        ret = TRUE;
+    }
+    
+    return ret;
+}
+
+void CheckIgnorePressure(BATTLECONTEXT *ctx, int battlerIdAttacker, int battlerIdTarget) {
+    if (battlerIdTarget != BATTLER_NONE && GetBattlerAbility(ctx, battlerIdTarget) == ABILITY_PRESSURE && ctx->battleMons[battlerIdAttacker].movePPCur[ctx->movePos[battlerIdAttacker]]) {
+        ctx->battleMons[battlerIdAttacker].movePPCur[ctx->movePos[battlerIdAttacker]]--;
+    }
+}
+
+BOOL ov12_022581BC(BattleSystem *bsys, BATTLECONTEXT *ctx) {
+    if (ov12_0223BFEC(bsys)) {
+        ctx->command = CONTROLLER_COMMAND_43;
+        return TRUE;
+    }
+    return FALSE;
+}
+
+int ov12_022581D4(BattleSystem *bsys, BATTLECONTEXT *ctx, int var, int battlerId) {
+    switch (var) {
+    case 0:
+        return ctx->fieldSideConditionFlags[BattleSystem_GetFieldSide(bsys, battlerId)];
+    case 1:
+        return ctx->fieldSideConditionData[BattleSystem_GetFieldSide(bsys, battlerId)].mistTurns;
+    case 2:
+        return ctx->selectedMonIndex[battlerId];
+    case 3:
+        return ctx->totalTurns;
+    case 4:
+        return ctx->levelUpMons;
+    case 5:
+        return ctx->safariRunAttempts;
+    case 6:
+        return ctx->totalTimesFainted[battlerId];
+    case 7:
+        return ctx->totalDamage[battlerId];
+    case 8:
+        return ctx->unk_21A8[battlerId][0];
+    case 9:
+        return ctx->unk_334.unk9C;
+    case 10:
+        return ctx->unk_3108;
+    case 11:
+        return ctx->unk_334.unkA4[battlerId];
+    case 12:
+        return ctx->unk_21A8[battlerId][2];
+    case 13:
+        return (int) ctx->command;
+    case 14:
+        return (int) ctx->commandNext;
+    }
+    
+    GF_ASSERT(FALSE);
+    return 0;
+}
+
+void ov12_022582B8(BattleSystem *bsys, BATTLECONTEXT *ctx, int var, int battlerId, int data) {
+    switch (var) {
+    case 0:
+        ctx->fieldSideConditionFlags[BattleSystem_GetFieldSide(bsys, battlerId)] = data;
+        break;
+    case 1:
+        ctx->fieldSideConditionData[BattleSystem_GetFieldSide(bsys, battlerId)].mistTurns = data;
+        break;
+    case 2:
+        ctx->selectedMonIndex[battlerId] = data;
+        break;
+    case 3:
+        ctx->totalTurns = data;
+        break;
+    case 9:
+        ctx->unk_334.unk9C = data;
+        break;
+    case 11:
+        ctx->unk_334.unkA4[battlerId] = data;
+        break;
+    default:
+        GF_ASSERT(FALSE);
+    }
+}
+
+extern const int ov12_0226CDCC[145];
+
+static int ov12_02258348(BATTLECONTEXT *ctx, int statChangeType, u32 flag) {
+    ctx->statChangeType = statChangeType;
+    ctx->statChangeParam = flag & (0x7FFFFF);
+    ctx->statChangeFlag = flag & (0xFF800000);
+    
+    if (flag & (1 << 30)) {
```

Comment:
enum/defines?

Reply:
I can prolly make it a define?

Reply:
not enough info to make it a define then?